File: solutions/1950-sign-of-the-product-of-an-array/solution.py
class Solution(object):
    def arraySign(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n=1
        for i in nums:
            n*=i
        if n==0:
            return 0
        elif n>0:
            return 1
        return -1

File: solutions/1950-sign-of-the-product-of-an-array/solution.java
class Solution {
    public int arraySign(int[] nums) {
        int n=1;
        for(int a:nums)
        {    
            if(a>0)
                n*=1;
            else if(a<0)
                n*=-1;
            else
                return 0;
        }
        return n;
    }
}
